root/dev_stats.py

```diff
@@ -0,0 +1,35 @@
+from collections import Counter
+
+dev_stats = {}
```

Comment:
you can use [`dataclass`](https://docs.python.org/3/library/dataclasses.html) - so it will clearly show structure that you described in docstring

and it's better to show docstring before variable